docs/serving/_index.md

```diff
@@ -11,7 +11,7 @@ The Knative Serving project provides middleware components that enable:
 
 - Rapid deployment of serverless containers.
 - Autoscaling including scaling pods down to zero.
-- Support for multiple networking layers such as Ambassador, Courier, Gloo, Istio, and Kong, for integration into existing environments.
+- Support for multiple networking layers such as Ambassador, Contour, Courier, Gloo, Istio, and Kong, for integration into existing environments.
```

Comment:
In addition to this change, could you please also fix `Courier` to [`Kourier`](https://github.com/knative-sandbox/net-kourier#kourier) ?

Reply:
Ah good catch :)